src/it/java/io/deephaven/benchmark/tests/standard/aggby/SortedComboTest.java

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2022-2023 Deephaven Data Labs and Patent Pending */
+package io.deephaven.benchmark.tests.standard.aggby;
+
+import org.junit.jupiter.api.*;
+import io.deephaven.benchmark.tests.standard.StandardTestRunner;
+
+/**
+ * Standard tests for the agg_by table operation. Applies aggregations to table data that require sorting
+ */
+public class SortedComboTest {
+    final StandardTestRunner runner = new StandardTestRunner(this);
+
+    @BeforeEach
+    public void setup() {
+        runner.tables("source");
+        runner.addSetupQuery("from deephaven import agg");
+    }
+
+    @Test
+    public void aggBySorted4Ops2Groups() {
+        var aggs = """
+        aggs = [
+           agg.median('Median=int250'), agg.pct(0.50, ['Percentile=int250']), 
+           agg.unique('Unique=int250'), agg.sorted_first('str250', ['int250'])
+        ]
+        """;
+        runner.addSetupQuery(aggs);
+
+        var q = "source.agg_by(aggs, by=['str250', 'str640'])";
+        runner.test("Sorted-AggBy- 4 Ops 2 Groups 160K Unique Vals", 160000, q, "str640", "str250", "int250");
+    }
+
+    @Test
+    public void aggBy3Medians2Groups() {
+        var aggs = """
+        aggs = [
+           agg.median('Median1=int250'), agg.median('Median2=int640'), agg.median('Median3=int1M')
+        ]
+        """;
+        runner.addSetupQuery(aggs);
+
+        var q = "source.agg_by(aggs, by=['str250', 'str640'])";
+        runner.test("Median-AggBy- 3 Medians 2 Groups 160K Unique Vals", 160000, q, "str640", "str250", "int250",
+                "int640", "int1M");
+    }
+
+    @Test
+    public void aggBy3Percentiles2Groups() {
+        var aggs = """
+        aggs = [
+           agg.pct(0.50, ['Percentile1=int250']), agg.pct(0.70, ['Percentile2=int640']), 
+           agg.pct(0.90, ['Percentile3=int1M'])
+        ]
+        """;
+        runner.addSetupQuery(aggs);
+
+        var q = "source.agg_by(aggs, by=['str250', 'str640'])";
+        runner.test("Percentile-AggBy- 3 Percentiles 2 Groups 160K Unique Vals", 160000, q, "str640", "str250",
+                "int250",
+                "int640", "int1M");
+    }
+
+    @Test
+    public void aggBy4FirstLastOps2Groups() {
```

Comment:
Do `first` and `last` do any sorting of the data?

Reply:
I don't think they do.  Moved then to a separate test class.